### PR TITLE
Correct casing for macOS

### PR DESCRIPTION
**Pull request by novialriptide**
source: https://www.apple.com/macos/sonoma/

### CODE REVIEW

**Comment by Starbuck5** [comment]
Are there other cases this should be changed?

**Comment by novialriptide** [comment]
Yeah, give me one second.

File: test/font_test.py [hunk 1]
```diff
@@ -79,12 +79,12 @@ def test_get_fonts_returns_something(self):
...
     # def XXtest_has_file_osx_10_5_sdk(self):
     #    import os
-    #    f = "/Developer/SDKs/MacOSX10.5.sdk/usr/X11/include/ft2build.h"
+    #    f = "/Developer/SDKs/macOSX10.5.sdk/usr/X11/include/ft2build.h"
     #    self.assertEqual(os.path.exists(f), True)
 
     # def XXtest_has_file_osx_10_4_sdk(self):
     #    import os
-    #    f = "/Developer/SDKs/MacOSX10.4u.sdk/usr/X11R6/include/ft2build.h"
+    #    f = "/Developer/SDKs/macOSX10.4u.sdk/usr/X11R6/include/ft2build.h"
```
Comment (ankith26) [inline test/font_test.py:87]: paths are case-sensitive, even though this is a comment, this change would be inaccurate

File: setup.py [hunk 2]
```diff
@@ -56,7 +56,7 @@
         "Operating System :: Microsoft :: Windows",
         "Operating System :: POSIX",
         "Operating System :: Unix",
-        "Operating System :: MacOS",
+        "Operating System :: macOS",
```
Comment (ankith26) [inline setup.py:59]: This is an incorrect tag. This must remain MacOS

**Review by MyreMylar — APPROVED** [review]
LGTM 👍

**Comment by MyreMylar** [comment]
Though for some reason the CI doesn't like it. I'm guessing that actual script variable name change is breaking something

**Comment by ankith26** [comment]
CI doesn't like it because the branch of this PR is not on this repo, and this change is triggering a docker image rebuild. I think we can safely ignore this fail, upon merging this PR, the error should not persist.

**Review by ankith26 — COMMENTED** [review]
:+1:

**Review by ankith26 — APPROVED** [review]

**Pull request merged**